docs/release-notes/rockcraft-1-14.rst

```diff
@@ -0,0 +1,160 @@
+container hosts.
+
+
+Minimum hardware requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- AMD64, ARM64, ARMv7-M, RISC-V 64-bit, PowerPC 64-bit little-endian, or S390x
+  processor
+- 2GB RAM
+- 10GB available storage space
+- Internet access for remote software sources and the Snap Store
+
+
+Platform requirements
+~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+  :header-rows: 1
+  :widths: 1 3 3
+
+  * - Platform
+    - Version
+    - Software requirements
+  * - GNU/Linux
+    - Popular distributions that ship with systemd and are `compatible with
+      snapd <https://snapcraft.io/docs/installing-snapd>`_
+    - systemd
+
+
+What's new
+----------
+
+Rockcraft 1.14 brings the following features, integrations, and improvements.
+
+New ``entrypoint-command`` project key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This key defines the rock's OCI entrypoint, bypassing the Pebble-powered entrypoint
+defined by Rockcraft. This can be necessary for cases like OS and base images, where
+an application-centric Pebble service is inadequate.
+
+See the :ref:`documentation <rockcraft-yaml-entrypoint-command>` for details.
+
+Minor features
+--------------
+
+Rockcraft 1.14 brings the following minor changes.
+
+Improved ``test`` command
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``test`` command now packs and tests all platforms that can be built on the running
+environment.
+
+Improved keyserver support in ``package-repositories``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If an attempt to connect to the default keyserver to retrieve a public key fails, the
+package-repositories mechanism will now try to connect through the standard port 80
+while also respecting proxy addresses set via the standard ``http_proxy`` and
+``https_proxy`` environment variables.
+
+New ``jlink-extra-modules`` key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This new :ref:`craft_parts_jlink_plugin` key allows defining additional modules that
+should be included in the generated OpenJDK image.
+
+Improved conflict detection between the Overlay and Build steps
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The collision resolution that takes place when the Stage step is executed now takes into
+account the contents of the Overlay.
+
+Improved documentation for 12-factor app extensions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The tutorial and reference pages for all 12-factor app extensions were improved
+based on user feedback, with fixes to multiple steps that were showstoppers.
+
+New ``maven-use`` plugin reference
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We've added a new :ref:`reference page <craft_parts_maven_use_plugin>`
+for the ``maven-use`` plugin.
+
+Clarified contribution guidelines
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We've added a `CONTRIBUTING.md`_ file describing the general approach and guidelines for
+contributing to the Rockcraft codebase. If you have an idea for an improvement, be sure
+to check it out.
```

Comment:
I'd like to add this to the new features, if that's OK with you.

Reply:
sure thing!